test/test_BEI.py

```diff
@@ -258,6 +338,33 @@ def test_methods_units(order, expected_unit, method_under_test, valid_constructo
     assert output.unit.is_equivalent(expected_unit)
```

Comment:
This test is not actually testing if the units are identical to what is expected.